spec/Signal/Cabin/Infotainment.vspec

```diff
@@ -117,3 +117,48 @@
   max: 250
   unit: m/s
   description: Vehicle speed, as sensed by the GPS receiver.
+
+- HMI:
+  type: branch
+  description: HMI related signals
+
+- HMI.CurrentLanguage:
+  type: string
+  description: ISO 639-1 standard language code for the current HMI
+  sensor: HMI
+
+- HMI.DateFormat:
+  type: string
+  enum: [ "DD MM YYYY", "MM DD YYYY" ]
```

Comment:
I find this inadequate.  It doesn't even include YYYY-MM-DD which is in fact an [ISO standard format](https://en.wikipedia.org/wiki/ISO_8601) and  preferred in various parts of Europe at least.  Yes, it is explicitly using hyphens.  

The proposal here with spaces does not seem not clarify the actual display format, which would depend on the characters used in between digits.   For example hyphens, or slashes:  DD/MM/YYYY.  Also it could be YYYY or only YY, and so on?

I'm not sure if it makes sense to include this in a standard signal database - it could be seen as an example, but if the intention is to make it usable for all cars then I am sure there are many more alternatives than those given.
Even if it is an example it should be as accurate as possible of course.